charts/pega/templates/_pega-deployment.tpl

```diff
@@ -160,7 +163,7 @@ spec:
         {{- $readinessProbe := .node.readinessProbe }}
         readinessProbe:
           httpGet:
-            path: "/prweb/PRRestService/monitor/pingService/ping"
+            path: "/{{ .root.Values.global.appContextRoot | default "prweb" }}/PRRestService/monitor/pingService/ping"
```

Comment:
Would it be better to set this in our `helpers` file so that we're not repeating things multiple times and instead we can just reference a single property? Just thinking more down the road when we might want to change things for some reason and might miss one of these hard-coded `prweb` values.

Reply:
Good idea @cdancy I will rework this to define/use a template in helpers with the app context setting from values (if that is not what you were proposing let me know)